fitbenchmarking/results_processing/tests/test_base_table.py

```diff
@@ -230,3 +233,21 @@ def test_display_str_both(self):
         self.table.options.comparison_mode = 'both'
         s = self.table.display_str([7, 9])
         self.assertEqual(s, '9 (7)')
+
+    @mock.patch("fitbenchmarking.results_processing.base_table.plt.savefig")
```

Comment:
We generally have 1 test file per source file and within, 1 class per function. Can you split this off into a new class.